src/layout/internal/dropper.ts

```diff
@@ -8,13 +8,20 @@ import SplitLayoutContainer from './SplitLayoutContainer';
 import {dropAble} from 'phovea_core/src/internal/dnd';
 import {EOrientation, IDropArea} from './interfaces';
 import {AParentLayoutContainer} from './AParentLayoutContainer';
+import LineUpLayoutContainer from './LineUpLayoutContainer';
```

Comment:
the lineup container doesn't put them next to each other but page wise. Is this change / option to be added in another PR?

Reply:
where should the widths of the different views be defined? or should they all get the same width defined somewhere in CSS?

Reply:
the interface already has an optional `minSize` information which can be used. For the rest, it should be the "natural" width/height of each view.